dart/lib/src/algorithms.dart

```diff
@@ -0,0 +1,106 @@
+// ignore_for_file: constant_identifier_names
+
+import 'package:snowball/src/snowball.dart';
+
+import '../ext/arabic_stemmer.dart';
+import '../ext/armenian_stemmer.dart';
+import '../ext/basque_stemmer.dart';
+import '../ext/catalan_stemmer.dart';
+import '../ext/danish_stemmer.dart';
+import '../ext/dutch_stemmer.dart';
+import '../ext/dutch_porter_stemmer.dart';
+import '../ext/english_stemmer.dart';
+import '../ext/esperanto_stemmer.dart';
+import '../ext/estonian_stemmer.dart';
+import '../ext/finnish_stemmer.dart';
+import '../ext/french_stemmer.dart';
+import '../ext/german_stemmer.dart';
+import '../ext/greek_stemmer.dart';
+import '../ext/hindi_stemmer.dart';
+import '../ext/hungarian_stemmer.dart';
+import '../ext/indonesian_stemmer.dart';
+import '../ext/irish_stemmer.dart';
+import '../ext/italian_stemmer.dart';
+import '../ext/lithuanian_stemmer.dart';
+import '../ext/nepali_stemmer.dart';
+import '../ext/norwegian_stemmer.dart';
+import '../ext/porter_stemmer.dart';
+import '../ext/portuguese_stemmer.dart';
+import '../ext/romanian_stemmer.dart';
+import '../ext/russian_stemmer.dart';
+import '../ext/serbian_stemmer.dart';
+import '../ext/spanish_stemmer.dart';
+import '../ext/swedish_stemmer.dart';
+import '../ext/tamil_stemmer.dart';
+import '../ext/turkish_stemmer.dart';
+import '../ext/yiddish_stemmer.dart';
```

Comment:
This file needs to be generated from `libstemmer/modules.txt` (see "Don't hardcode algorithm names" in `CONTRIBUTONG.rst`).

Reply:
Hmm, it looks like there's actually already a script to generate it in your patch, so I think you just need to remove this file from git and add it to `.gitignore`.

Reply:
That's fair. I originally had in mind having some substitution magic similar to `SNOWBALL_VERSION`, perhaps `SNOWBALL_LANGUAGES`, that would allow applying that substitution to an existing file that is checked in. However, technically if the entire file is generated it doesn't need to be checked in.

I've made the change.